-standup-template.md

```diff
@@ -2,10 +2,11 @@
 
 <sup>[from template][standup-template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
 
-Time:      ...  
-Location:  https://meet.jit.si/hyphacoop  
-Attending: ...  
-Notetaker: :raising_hand: el, yj, gi, pc, dc, uv, bl  
+Time:       ...  
+Location:   https://meet.jit.si/hyphacoop  
```

Comment:
Why are these in the diff 🤔

Reply:
Because you'd aligned content when you tweaked template, and adding "cultivator" makes the indent one space deeper